src/main/scala/chisel3/ltl/LTL.scala

```diff
@@ -6,6 +6,7 @@ import chisel3._
 import chisel3.util.circt._
 import chisel3.experimental.hierarchy.{Instance, Instantiate}
 import chisel3.experimental.SourceInfo
+import firtoolresolver.shaded.coursier.ivy.PropertiesPattern.ChunkOrProperty.Prop
```

Comment:
Why this is required?

Reply:
Probably an accidental VSCode import. It loves pulling things from `firtoolresolver.shaded`...
```suggestion
```

Reply:
stupid metals, constantly auto-importing random stuff